packages/api-providers/src/ipfs/substrate/vanchor.ts

```diff
@@ -1,9 +1,10 @@
 import { getCachedFixtureURI, withLocalFixtures } from '@webb-dapp/api-providers/utils';
 
-export async function fetchSubstrateVAnchorProvingKey() {
+export async function fetchSubstrateVAnchorProvingKey(edges: number = 2, abortSignal: AbortSignal) {
```

Comment:
Remove default parameter 2. Also, the parameter for web3 equivalent is maxEdges (i.e. a bridge between two chains has maxEdges == 1).

Reply:
it has a default value  as a place holder we should add support for other sizes on another PR

Reply:
Can we remove the default parameter though? It's never actual default because the abortSignal in the function signature is required.